### PR TITLE
Added change-password.html and change-password.css.

### PR DESCRIPTION
**Pull request by Greengrer**

### CODE REVIEW

**Review by nomarchia — APPROVED** [review]

**Review by AlexKievDeveloper — APPROVED** [review]

**Review by RomanKlishch — APPROVED** [review]

File: src/main/webapp/WEB-INF/templates/change-password.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Create a greeting card!</title>
+		<meta name="viewport" content="width=device-width, initial-scale=0.86, maximum-scale=3.0, minimum-scale=0.86" />
+		<link rel="stylesheet" href="../../assets/css/reset.css" />
+		<link rel="stylesheet" href="../../assets/css/common.css" />
+		<link rel="stylesheet" href="../../assets/css/change-password.css" />
+		<meta charset="UTF-8">
+	</head>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/change-password.html:10]: Common links must be in common file. It uses in all html pages.

File: src/main/webapp/WEB-INF/templates/change-password.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		<div class="wrapper">
+			<div class="main-body">
+				<header class="header">
+					<div class="app-name">App-name</div>
+					<div class="navigation-row">
+						<a href="#" class="navigation-link">Home</a>
+						<a href="#" class="navigation-link">Your cards</a>
+						<a href="#" class="navigation-link">Profile</a>
+						<a href="#" class="navigation-link">Log out</a>
+					</div>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/change-password.html:21]: Navigation menu must be in common file. Nav menu uses in all pages.

File: src/main/webapp/WEB-INF/templates/change-password.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+							<input type="password" id="old-password" class="styled-as-input" name="password" placeholder="put your old password here">
+							<input type="password" id="new-password" class="styled-as-input" name="new-password" placeholder="put your new password here">
+							<input type="password" id="new-password-check" class="styled-as-input" name="new-password-check" placeholder="and here, to check">
+							<input type="submit" class="button-link" value="Log in">
+						</form>
+					</div>
+				<footer class="footer">
+					<div>Some words</div>
+					<div>Copyright, 2020</div>
+				</footer>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/change-password.html:34]: Footer must be in common file. Footer uses in all pages.

File: src/main/webapp/WEB-INF/templates/change-password.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+						<a href="#" class="navigation-link">Profile</a>
+						<a href="#" class="navigation-link">Log out</a>
+					</div>
+				</header>
+					<div class="body__column">
+						<form action="" class="profile-text">
+							<input type="password" id="old-password" class="styled-as-input" name="password" placeholder="put your old password here">
+							<input type="password" id="new-password" class="styled-as-input" name="new-password" placeholder="put your new password here">
+							<input type="password" id="new-password-check" class="styled-as-input" name="new-password-check" placeholder="and here, to check">
+							<input type="submit" class="button-link" value="Log in">
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/change-password.html:28]: I think it must be button with name Change. We change password.

File: src/main/webapp/WEB-INF/templates/profile.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Create a greeting card!</title>
+		<meta name="viewport" content="width=device-width, initial-scale=0.86, maximum-scale=3.0, minimum-scale=0.86" />
+		<link rel="stylesheet" href="../../assets/css/reset.css" />
+		<link rel="stylesheet" href="../../assets/css/common.css" />
+		<link rel="stylesheet" href="../../assets/css/profile.css" />
+		<meta charset="UTF-8">
+	</head>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/profile.html:10]: Common links must be in common file. It uses in all html pages.

File: src/main/webapp/WEB-INF/templates/profile.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+	<body>
+		<div class="wrapper">
+			<div class="main-body">
+				<header class="header">
+					<div class="app-name">App-name</div>
+					<div class="navigation-row">
+						<a href="#" class="navigation-link">Home</a>
+						<a href="#" class="navigation-link">Your cards</a>
+						<a href="#" class="navigation-link">Log out</a>
+					</div>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/profile.html:20]: Navigation menu must be in common file. Nav menu uses in all pages.

File: src/main/webapp/WEB-INF/templates/profile.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+								</div>
+							</form>
+							<a href="#" class="button-link">Change password</a>
+						</div>
+					</div>
+				<footer class="footer">
+					<div>Some words</div>
+					<div>Copyright, 2020</div>
+				</footer>
+			</div>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/profile.html:43]: Footer must be in common file. Footer uses in all pages.

File: src/main/webapp/WEB-INF/templates/change-password.html (hunk 4 above)
Comment (Greengrer) [reply src/main/webapp/WEB-INF/templates/change-password.html:28]: Oooohh, thanks, I absolutely missed that part.

**Pull request closed without merging**